css/properties/justify-items.json

```diff
@@ -79,6 +79,41 @@
             }
           }
         },
+        "block_context": {
+          "__compat": {
+            "description": "Supported in Block Layout",
+            "spec_url": [
+              "https://drafts.csswg.org/css-align/#justify-items-property",
+              "https://drafts.csswg.org/css-align/#justify-block"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "130"
```

Comment:
You can add Chrome CL: https://chromiumdash.appspot.com/commit/f47b3fe44349130682e544e6a372b581c5714996